core/sessions/oidcauth/oidc.go

```diff
@@ -0,0 +1,675 @@
+/*
+The OIDC module handles authentication by redirecting to a
+Open ID Connect Identity Provider.
+*/
+package oidcauth
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/subtle"
+	"database/sql"
+
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+	"time"
```

Comment:
I would expect the linter to complain about this, but maybe it just cares about sorting within each block? We typically keep std lib as one block though:
```suggestion
	"context"
	"crypto/rand"
	"crypto/subtle"
	"database/sql"
	"encoding/base64"
	"errors"
	"fmt"
	"io"
	"log"
	"net/http"
	"strings"
	"time"
```

Reply:
👍